ui/jstests/test_manage_taxonomies.jsx

```diff
@@ -45,14 +45,10 @@ define(['QUnit', 'jquery', 'manage_taxonomies', 'reactaddons',
    * */
   function  assertAddTermCommon(assert, component) {
     // check term title
-    var listOfLinks = React.addons.TestUtils.
-      scryRenderedDOMComponentsWithTag(
-        component,
-        'a'
-    );
-    assert.equal(listOfLinks.length, 2);
-    var linkHeader = React.findDOMNode(listOfLinks[0]);
-    assert.equal(linkHeader.innerHTML, vocabulary.name);
+    var node = React.findDOMNode(component);
+    var $vocabTitles = $(node).find(".vocab-title");
+    assert.equal($vocabTitles.length, 1);
+    assert.equal($vocabTitles[0].innerHTML, vocabulary.name);
```

Comment:
it is jquery,  why dont you use **linkHeader.html()** instead of **linkHeader.innerHTML** ?